cmd/ctr/inspect.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	gocontext "context"
+	"fmt"
+
+	"github.com/docker/containerd/api/execution"
+	"github.com/urfave/cli"
+)
+
+var inspectCommand = cli.Command{
```

Comment:
Can you remove all the whitespace inbetween lines so that it matches your other file?